pepper-apis/src/main/java/org/broadinstitute/ddp/service/PdfService.java

```diff
@@ -126,6 +126,6 @@ public String generateAndUpload(Handle handle,
                 participantGuid,
                 pdfConfiguration.getConfigName(),
                 pdfConfiguration.getVersion().getVersionTag(),
-                new ByteArrayInputStream(pdf));
+                pdfStream);
```

Comment:
Should we `close()` `pdfStream` prior to return?

Reply:
Yes we should. Thanks! d0afd6bd